client/modules/sheaf-core/src/extensions/spellcheck/spellchecker/normalize.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Pairs of strings, with the first string being text to replace with the
+ * latter. Used for normalizing text.
+ */
+const REPLACEMENTS: [string, string][] = [
```

Comment:
Add a comment/todo about how this is latin-centric?

Reply:
My hope with this specific object is that it's just for replacing unusual substitutes for characters - I don't think there should be any language specific replacements, or if there is, it's language-specific enough that it doesn't conflict with anything.